apis/nucleus/src/index.js

```diff
@@ -446,13 +446,14 @@ function nuked(configuration = {}) {
            * @param {boolean=} [options.dense=false] Reduces padding and text size (not applicable for existing objects)
            * @param {string=} [options.stateName="$"] Sets the state to make selections in (not applicable for existing objects)
            * @param {object=} [options.properties={}] Properties object to extend default properties with
+           * @returns {Promise<void>} A promise that resolves when the data is fetched.
            *
            * @since 1.1.0
            * @instance
            * @example
            * fieldInstance.mount(element);
            */
-          mount(element, options = {}) {
+          async mount(element, options = {}) {
```

Comment:
Needs a spec update. Normally a very breaking change, but as we didn't return anything before it should be quite safe.

Reply:
Updated ✅